sources/tools/Stride.TextureConverter.Wrappers/FreeImage.Net/FreeImage.NET.csproj

```diff
@@ -14,7 +14,6 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <DocumentationFile>
     </DocumentationFile>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
This `<TreatWarningsAsErrors>` is not an AssemblyProcessor option, but a C# compiler option.

Reply:
My mistake, Changed now 😅